src/WebJobs.Script/Extensions/HttpRequestExtensions.cs

```diff
@@ -87,7 +86,7 @@ public static bool IsAppServiceInternalRequest(this HttpRequest request, IEnviro
             // this header will *always* be present on requests originating externally (i.e. going
             // through the Anatares front end). For requests originating internally it will NOT be
```

Comment:
typo

Reply:
This was a previously existing comment (not part of this PR), but fixed.